MonMaraicher/Features/Map/MapViewService.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MapViewService.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 15/03/2024.
+//
+
+import Foundation
+
+final class MapViewService {
```

Comment:
The name of the service is to specific IMO. We could easily imagine that another screen could need that same data (farmers). So I would name it FarmerService instead. A service which provide and manage Farmer data. What do you think?

Reply:
You cannot mock your service here because youre not using protocol

Reply:
What do you mean by: mock your service?

Reply:
Is using a struct in this case better?

Reply:
If you need to test your map VM for example, you will have trouble to test it because your service is not mockable